modules/sms_blast/src/Tests/SmsBlastWebTest.php

```diff
@@ -60,16 +60,28 @@ protected function setUp() {
    * Tests sending SMS blast.
    */
   public function testSendBlast() {
-    // Create users with two phone numbers. Only one message should be sent to
-    // each user.
-    $phone_numbers = ['+123123123', '+456456456'];
-    for ($i = 0; $i < 3; $i++) {
-      // Create an unverified user.
-      $this->createEntityWithPhoneNumber($this->phoneNumberSettings, $phone_numbers);
-      // Create a verified user.
-      $entity = $this->createEntityWithPhoneNumber($this->phoneNumberSettings, $phone_numbers);
-      $this->verifyPhoneNumber($entity, $phone_numbers[0]);
+    // Create users with multiple phone numbers. Only one message should be sent
+    // to each user.
+    $phone_numbers = $this->randomPhoneNumbers();
+    $entities = [];
+    for ($i = 0; $i < 6; $i++) {
+      /** @var \Drupal\user\UserInterface $user */
+      $user = $this->createEntityWithPhoneNumber($this->phoneNumberSettings, $phone_numbers);
+      // Need to activate so when DER does entity validation it is included by the
+      // UserSelection plugin.
+      $user->activate()->save();
```

Comment:
The patch already addresses the few instances this is needed. You only need to activate the sender/recipient user if you are validating the sms message. Validation is currently only happening in `provider::queue` and `accountregistration` services.

Note: Validation does not occur as a part of saving entities, it must be done as an explicit step.